liferay-echo/liferay-portal
# LPS-The exported zip of utility page should use utility page name

## Motivation

[Link to task](https://issues.liferay.com/browse/LPS-172869)

Utility pages are exported with their external reference code in the filename. In case of fragments and page templates, we generated them by hand, in utility pages it's the UUID.

The goal of the task is to standardise this so that utility pages have the same type of external reference code as fragments and layout page templates

## Change summary:

* When we don't receive an ERC, we autogenerate them just like with fragments and page templates
* Integration tests


## Test Scenario

* Create a new utility page
* Export it - the name of the utility page should be present in the zip file name
